### PR TITLE
Fixed Edit on github links for v13.x documentation 

### PR DESCRIPTION
**Pull request by javad94**
### Checklist for PRs

- [ ] Added `.. versionadded:: version`, `.. versionchanged:: version` or `.. deprecated:: version` to the docstrings for user facing changes (for methods/class descriptions, arguments and attributes)
- [ ] Created new or adapted existing unit tests
- [ ] Documented code changes according to the [CSI standard](https://standards.mousepawmedia.com/en/stable/csi.html)
- [x] Added myself alphabetically to `AUTHORS.rst` (optional)
- [ ] Added new classes & modules to the docs and all suitable `__all__` s

### CODE REVIEW

**Comment by javad94** [comment]
btw jinja2 gives ImportError when using `sphinx-build docs/source docs/build/html` command. Downgrading jinja2 from `3.1.2` to `3.0.3` fixed this issue. So should I update [requirements-docs.txt](https://github.com/python-telegram-bot/python-telegram-bot/blob/v13.x/docs/requirements-docs.txt) too?
```
Traceback (most recent call last):
  File "C:\Users\Javad\Anaconda3\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\Javad\Anaconda3\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "D:\python-projects\ptb\venv\Scripts\sphinx-build.exe\__main__.py", line 4, in <module>
  File "D:\python-projects\ptb\venv\lib\site-packages\sphinx\cmd\build.py", line 25, in <module>
    from sphinx.application import Sphinx
  File "D:\python-projects\ptb\venv\lib\site-packages\sphinx\application.py", line 43, in <module>
    from sphinx.registry import SphinxComponentRegistry
  File "D:\python-projects\ptb\venv\lib\site-packages\sphinx\registry.py", line 24, in <module>
    from sphinx.builders import Builder
  File "D:\python-projects\ptb\venv\lib\site-packages\sphinx\builders\__init__.py", line 26, in <module>
    from sphinx.util import import_object, logging, progress_message, rst, status_iterator
  File "D:\python-projects\ptb\venv\lib\site-packages\sphinx\util\rst.py", line 21, in <module>
    from jinja2 import Environment, environmentfilter
ImportError: cannot import name 'environmentfilter' from 'jinja2' (D:\python-projects\ptb\venv\lib\site-packages\jinja2\__init__.py)
```

**Review by harshil21 — APPROVED** [review]
Thanks for the fix. Is your sphinx version the one specified in requirements-docs.txt? The error probably happens since the versions aren't exactly matching the ones we specified.

**Comment by javad94** [comment]
> Thanks for the fix. Is your sphinx version the one specified in requirements-docs.txt? The error probably happens since the versions aren't exactly matching the ones we specified.

Yes, it is the same version.
image

**Comment by harshil21** [comment]
okay since I see you're on a venv you probably have the right versions I'd say to go ahead and add that in (Jinja2 < 3.1). Makes sense you're getting this since I see [this](https://github.com/sphinx-doc/sphinx/blob/v3.5.4/setup.py#L24) in the sphinx source code.

**Comment by javad94** [comment]
> okay since I see you're on a venv you probably have the right versions I'd say to go ahead and add that in (Jinja2 < 3.1). Makes sense you're getting this since I see [this](https://github.com/sphinx-doc/sphinx/blob/v3.5.4/setup.py#L24) in the sphinx source code.

Of course, done.

File: docs/requirements-docs.txt [hunk 1]
```diff
@@ -4,3 +4,4 @@ sphinx-pypi-upload
 # More instructions at source/_static/dark.css
 # Ofc once https://github.com/readthedocs/sphinx_rtd_theme/issues/224 is closed, we should use that
 sphinx_rtd_theme==0.5.2
+Jinja2<3.1
```
Comment (harshil21) [inline docs/requirements-docs.txt:7]: can also add a comment here saying why

Comment (javad94) [reply docs/requirements-docs.txt:8]: Fair enough. Is the newly added comment okay?

File: docs/requirements-docs.txt [hunk 2]
```diff
@@ -4,3 +4,5 @@ sphinx-pypi-upload
 # More instructions at source/_static/dark.css
 # Ofc once https://github.com/readthedocs/sphinx_rtd_theme/issues/224 is closed, we should use that
 sphinx_rtd_theme==0.5.2
+# Restricted to fix ImportError for v3.1 and higher of Jinja2 alongside sphinx v3.5.4 as discussed in https://github.com/python-telegram-bot/python-telegram-bot/pull/3164#issuecomment-1193002217
```
Comment (harshil21) [inline docs/requirements-docs.txt:7]:
```suggestion
# Restricted to fix ImportError for v3.1 and higher of Jinja2 alongside sphinx v3.5.4 as discussed in 
# https://github.com/python-telegram-bot/python-telegram-bot/pull/3164#issuecomment-1193002217
```
just a nitpick to put it in a new line :)

Comment (javad94) [reply docs/requirements-docs.txt]: make sense :)

**Comment by javad94** [comment]
@harshil21 Anything else needed to merge this PR?

**Comment by harshil21** [comment]
Thanks for the contribution!

**Comment by javad94** [comment]
> Thanks for the contribution!

You're very welcome.

**Pull request merged**